pkg/stanza/adapter/converter.go

```diff
@@ -59,6 +59,9 @@ type Converter struct {
 	pLogsChan chan plog.Logs
 
 	stopOnce sync.Once
+
+	// stopChan is an internal communication channel signalling stop was called
```

Comment:
It looks like this is only to signal the converter to stop, so we could rename it `converterChan` to be a bit more clear.